lib/parsePackageJSON.js

```diff
@@ -0,0 +1,14 @@
+import path from "path";
+import { readFileSync } from "fs";
+
+export default function buildFromPackageJSON() {
```

Comment:
It's a bit unusual that the filename is parsePackageJSON but the function name is buildFromPackageJSON. I would prefer not using a default export here so that it's not as easy to end up with different names for the same function like this. Also, since this function isn't building the output directory, I think it would make more sense to have a different name (one without the word `build` in it)

Last bit - I actually wouldn't mind this function just being inlined into the main file rather than having its own file. As the project grows, it may become more needed to split it up into more files but for now I think another 7 lines of code to it is fine.

```suggestion
export function packageConfigFromPackageJson() {
```

Reply:
Changed name and moved to main file.